tests/data/inference/test_aggregator.py

```diff
@@ -33,23 +35,32 @@ def aggregate(self, mode, fixture):
         self.assertTensorEqual(output, fixture)
 
     def test_overlap_crop(self):
-        fixture = torch.Tensor((
+        fixture = (
             (0, 0, 2, 2),
             (0, 0, 2, 2),
             (4, 4, 6, 6),
             (4, 4, 6, 6),
-        )).reshape(1, 1, 4, 4)
+        )
         self.aggregate('crop', fixture)
 
     def test_overlap_average(self):
-        fixture = torch.Tensor((
+        fixture = (
             (0, 1, 1, 2),
             (2, 3, 3, 4),
             (2, 3, 3, 4),
             (4, 5, 5, 6),
-        )).reshape(1, 1, 4, 4)
+        )
         self.aggregate('average', fixture)
 
+    def test_overlap_hann(self):
+        fixture = (
+            ( 0 / 3,  2 / 3,  4 / 3,  6 / 3),  # noqa: E201, E241
+            ( 4 / 3,  6 / 3,  8 / 3, 10 / 3),  # noqa: E201, E241
+            ( 8 / 3, 10 / 3, 12 / 3, 14 / 3),  # noqa: E201, E241
+            (12 / 3, 14 / 3, 16 / 3, 18 / 3),
```

Comment:
I've reformatted this so it -hopefully- a bit more interpretable.

How did you compute these values?

Reply:
Agree with the improved interpretability.
I've computed the 2D window function and the resulting values by hand, by using the initial 1d functions given by PyTorch as a starting point.